docs/source/guides/using_dask_entitysets.ipynb

```diff
@@ -0,0 +1,245 @@
+    "\n",
+    "When using a pandas DataFrames, the ordering of the underlying DataFrame rows is maintained by Featuretools. For a Dask DataFrame, the ordering of the DataFrame rows is not guaranteed, and Featuretools does not attempt to maintain row order. If ordering is important, close attention must be paid to any output to avoid issues.\n",
+    "\n",
+    "### EntitySet Limitations\n",
+    "\n",
+    "When creating a Featuretools `EntitySet` that will be made of Dask DataFrames, all of the DataFrames used to create the `EntitySet` must be of the same type, either all Dask DataFrames or all pandas DataFrames. Featuretools does not support creating an `EntitySet` containing a mix of Dask and pandas DataFrames.\n",
+    "\n",
+    "Additionally, the `EntitySet.add_interesting_values()` method is not supported when using a Dask `EntitySet`. If needed, users can manually set `interesting_values` on entities by assigning them directly with syntax similar to this: `es[\"dataframe_name\"].ww[\"variable_name\"].ww.metadata[\"interesting_values\"] = [\"Value 1\", \"Value 2\"]`.\n",
+    "\n",
+    "\n",
+    "### DFS Limitations\n",
+    "\n",
+    "There are a few key limitations when generating a feature matrix from a Dask `EntitySet`.\n",
+    "\n",
+    "If a `cutoff_time` parameter is passed to `featuretools.dfs()` it should be a single cutoff time value, or a pandas DataFrame. The current implementation will still work if a Dask DataFrame is supplied for cutoff times, but a `.compute()` call will be made on the DataFrame to convert it into a pandas DataFrame. This conversion will result in a warning, and the process could take a considerable amount of time to complete depending on the size of the supplied DataFrame.\n",
+    "\n",
+    "Additionally, Featuretools does not currently support the use of the `approximate` or `training_window` parameters when working with Dask EntitySets, but should in future releases.\n",
+    "\n",
+    "Finally, if the output feature matrix contains a boolean column with `NaN` values included, the column type may have a different datatype than the same feature matrix generated from a pandas `EntitySet`.  If feature matrix column data types are critical, the feature matrix should be inspected to make sure the types are of the expected types, and recast as necessary.\n",
+    "\n",
+    "### Other Limitations\n",
+    "\n",
+    "In some instances, generating a feature matrix with a large number of features has resulted in memory issues on Dask workers. The underlying reason for this is that the partition size of the feature matrix grows too large for Dask to handle as the number of feature columns grows large. This issue is most prevalent when the feature matrix contains a large number of columns compared to the DataFrames that make up the entities. Possible solutions to this problem include reducing the partition size used when creating the DataFrames or increasing the memory available on Dask workers.\n",
```

Comment:
"compared to the DataFrames that make up the entities" -> "compared to the DataFrames in the EntitySet"